data/json/monsters/turrets.json

```
@@ -55,20 +85,20 @@
     "armor_cut": 16,
     "vision_night": 3,
     "revert_to_itype": "bot_antimateriel",
-    "starting_ammo": { "50bmg": 100 },
+    "starting_ammo": { "50bmg": 400 },
     "special_attacks": [
       {
         "type": "gun",
         "cooldown": 1,
         "move_cost": 150,
-        "gun_type": "m107a1",
+        "gun_type": "m2browning",
         "ammo_type": "50bmg",
         "fake_skills": [ [ "gun", 8 ], [ "rifle", 8 ] ],
         "fake_dex": 12,
-        "ranges": [ [ 0, 40, "DEFAULT" ] ],
+        "ranges": [ [ 0, 10, "AUTO" ], [ 0, 40, "DEFAULT" ] ],
```

This means that it will go full-auto on ranges 0-10 AND it will fire its standard fire mode on ranges 0-40 which is obviously overlaps with range 0-10.
```suggestion
        "ranges": [ [ 0, 10, "AUTO" ], [ 11, 40, "DEFAULT" ] ],
```

---

Missed that when I wrote it. It was meant to be 11-40.